### DISCUSSION TITLE
Can't resolve `react-intl`

### DISCUSSION THREAD

**Discussion by ar7casper** [gatsbyjs/themes#119] [Q&A]
In the context of a monorepo, I get the following error.
react-intl is installed and shows up on `node_modules`

> Generating SSR bundle failed
>
> Can't resolve 'react-intl' in '.../monorepo/node_modules/gatsby-theme-i18n-react-intl/src'
> 
> If you're trying to use a package make sure that 'react-intl' is installed. If you're trying to use a local file make sure that the path is correct.
> 
> File: ../../node_modules/gatsby-theme-i18n-react-intl/src/wrap-page-element.js

**Comment by LekoArts** [comment]
If you use multiple versions of `react-intl` you might need to use `resolutions` to force a single version.
